### DISCUSSION TITLE
Error CLI Vue

### DISCUSSION THREAD

**Discussion by Jorge-Macias-Cruz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#487] [Q&A] [answered]
a la hora de ejecutar el comando : npm install -g @vue/cli
me aparece esto Tengo MacOS

![Captura de pantalla 2022-03-22 a la(s) 15 16 37](https://user-images.githubusercontent.com/37346944/159577711-0e66f1bc-ef63-4d70-9774-5f8fed3c64ef.png)

**Comment by gersondlarosa** [comment]
Ya instalaste Node.js recuerda que hay que tener instalado Node.js

**Reply by Jorge-Macias-Cruz** [reply]
ya compañero

**Reply by gersondlarosa** [reply]
eso que dice Practica 5 Macias es una carpeta o tu lo escribiste

**Reply by Jorge-Macias-Cruz** [reply]
no ya lo corri igual en mi terminal y me sigue apareciendo los mismos errores

**Comment by doguedogue** [comment, marked as answer]
Usa sudo para instalar precediendo la instrucción npm ...

**Reply by Jorge-Macias-Cruz** [reply]
Cierto, gracias ya me salen los "warning" pero creo eso no afecta
